main/app/attentionRelated/ears/conf/client.ini

```diff
@@ -2,3 +2,4 @@ subsystems          (recog)
 reachingRangeMin    (-0.5 -0.3 -0.15)
 reachingRangeMax    (-0.1 0.3 0.5)
 robot               icub
+zOffset             0.0
```

Comment:
Rather than introducing yet another parameter to be set in various modules, I have to say that I would prefer a hard-coded `zOffset` within `subSystem_Karma` rather than cluttering the config files.